frigate/config.py

```diff
@@ -128,11 +128,30 @@ def validate_password(cls, v, values):
         return v
 
 
+class PtzAutotrackConfig(FrigateBaseModel):
+    enabled: bool = Field(default=False, title="Enable PTZ object autotracking.")
+    track: List[str] = Field(default=DEFAULT_TRACKED_OBJECTS, title="Objects to track.")
+    required_zones: List[str] = Field(
+        default_factory=list,
+        title="List of required zones to be entered in order to begin autotracking.",
+    )
+    return_preset: str = Field(
```

Comment:
This either needs a default value or needs to be optional, otherwise the config fails to start for cameras that are not using this feature

Reply:
Gotcha, was a little unsure how required/optional worked for subsections in the config.

So should everything under the `autotracking:` section be optional then?

Reply:
`Optional` is only used for fields that need to have a default of null and get set somewhere else. You should just create a default value for them in this case